app/lib/providers/bangumi/search/BangumiSearchService.dart

```diff
@@ -47,15 +47,15 @@ class BangumiSearchService {
     /// does exist some results, so we need to set [chii_searchDateLine] cookie
     /// header here
     /// see https://github.com/bangumi/api/issues/43
-    String requestUrl =
+    String requestUrlConst =
```

Comment:
this is actually not a const, see https://dart.dev/guides/language/language-tour#final-and-const

can we just call it `requestUrl`?